README.md

```diff
@@ -25,19 +25,31 @@ RandomizedSearchCV was applied for hyperparameter optimization.
 The model was evaluated using metrics such as accuracy, precision, recall, and F1-score.
 
 ## Usage
-For the first time running the project, create the conda environment by running the following in the root of the repository:
 
-```bash
-conda-lock install --name wine-quality-regressor conda-lock.yml
```

Comment:
Okay, that makes sense. I can add that as the second option should Docker container fail to launch.

Reply:
@sgdkd I thought about it some more and I realized that that second option will only work on Linux machines because the multi-platform conda-lock.lock is gonna be deleted in #36. I recommend an instruction to fall back on `environment.yml` since it stays consistent and can work with any platform. Please tell me what you think about this change.

Reply:
Sounds good!